src/views/studio/MyVideosView/MyVideosView.tsx

```diff
@@ -238,6 +265,20 @@ export const MyVideosView = () => {
           <>
             <TabsContainer>
               <Tabs initialIndex={0} tabs={mappedTabs} onSelectTab={handleSetCurrentTab} />
+              <SortContainer>
+                <Text variant="body2">Sort by</Text>
+                <Select
```

Comment:
We should be able to change height of that element. I know it has 50px everywhere in forms, but in this case it has 40px, according to design

Reply:
It seems in the design selects now have two sizing options. Would it be a lot of work to add that?

Reply:
That should be in a different PR no?